### ISSUE TITLE
AWS

### ISSUE THREAD

**Issue by evgenymun** [evgenymun/kaggleSpringLeaf#1]
My laptop doesn't have enough memory to read in the train.csv. We need to install anaconda and spark on AWS. 
I ran this code and with no result: 

import pandas as pd
train = pd.read_csv("input/train.csv")
print('\nSummary of train dataset:\n')
print(train.describe())

It looks like i need at least 8GB which i don't have.

**Comment by TalkData2Me** [comment]
Did you try the edX VM?

**Comment by evgenymun** [comment]
Yes I tried on vm and regular installation. Vm is running on local resources. I turned off wifi and it worked.
